secator/tasks/trufflehog.py

```
@@ -83,20 +83,15 @@ class trufflehog(Command):
     }
     output_types = [Tag, Info]
     ignore_return_code = True
-    install_pre = {
-        'apt': ['git', 'golang'],
-        'apk': ['git', 'go'],
-        'pacman': ['git', 'go'],
-        '*': ['git', 'go']
-    }
     install_version = 'v3.91.0'
+    install_pre_cmd = {'apt': ['git', 'golang'], '*': ['git', 'go']}
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)